project/AppDependencies.scala

```diff
@@ -25,11 +25,13 @@ object AppDependencies {
     "uk.gov.hmrc"   %% "bootstrap-play-26" % "0.34.0",
     "uk.gov.hmrc"   %% "domain"            % "5.6.0-play-26",
     "uk.gov.hmrc"   %% "play-hmrc-api"     % "3.4.0-play-26",
-    "org.typelevel" %% "cats-core"         % "1.6.0"
+    "org.typelevel" %% "cats-core"         % "1.6.0",
+    "com.chuusai"   %% "shapeless"         % "2.3.3"
```

Comment:
Is that ok to use these dependencies? We haven't used, don't know why?

Reply:
They are used for enum json format generation (and shapeless is used in other HMRC projects).

Reply:
if it already used then we are fine :)